src/darwin/Framework/CHIP/MTRDeviceController.mm

```diff
@@ -93,6 +95,9 @@ @interface MTRDeviceController ()
 @property (readonly) MTRDeviceControllerFactory * factory;
 @property (readonly) NSMutableDictionary * nodeIDToDeviceMap;
 @property (readonly) os_unfair_lock deviceMapLock; // protects nodeIDToDeviceMap
+@property (readonly) chip::Dnssd::ResolverProxy mDNSResolver;
+@property (readonly) MTRDeviceDiscoveryDelegateBridge * deviceDiscoveryDelegateBridge;
+@property (readonly) chip::Dnssd::ResolverDelegateProxy resolverDelegagteProxy;
```

Comment:
Spelling?  Also, where is this used?

Reply:
I removed all discovery and ethernet pairing changes because they are useful for test but not a mandatory feature for Matter certification.